AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -289,6 +289,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/101665
+ethereumfaucet.info#%#//scriptlet("set-constant", "tr", "1")
```

Comment:
This rule works, please change your rule to this. 
`ethereumfaucet.info#%#//scriptlet("adjust-setInterval", "ct", "1000", "0.02")`

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/101689/commits/07c2400c89a97bb9751789e5102200d777d9eb0c